packages/react-i18n/src/i18n.ts

```diff
@@ -214,16 +224,41 @@ export default class I18n {
   }
 
   private humanizeDate(date: Date, options?: Intl.DateTimeFormatOptions) {
-    const today = new Date();
-
-    if (isSameDate(today, date)) {
-      return this.translate('today');
+    const time = this.formatDate(date, {
+      ...options,
+      style: DateStyle.Time,
+    }).toLocaleLowerCase();
+
+    if (isLessThanOneMinuteAgo(date)) {
+      return this.translate('humanize.now');
+    } else if (isLessThanOneHourAgo(date)) {
```

Comment:
Each one of these `else if`s is a return so you don't need to make them `else if` they can be standalone early return `if` statements. This makes it slightly easier to parse and means you can move the `time` assignment further down so it's only assigned when needed.